src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ThrowableToErrorMapper.java

```diff
@@ -69,6 +70,17 @@ public final class ThrowableToErrorMapper {
           return handleDriverException((DriverException) throwable, message);
         }
 
+        // handle an invalid Content-Type header
+        if (throwable instanceof NotSupportedException
+            && throwable
+                .getMessage()
+                .contains("The content-type header value did not match the value")) {
```

Comment:
Is this check needed? Wouldn't we always convert this exception to 415?

Reply:
Yes, let me change that.
I was just try to make sure.